mis_builder/models/mis_report_instance.py

```diff
@@ -75,11 +75,13 @@ class MisReportInstancePeriod(models.Model):
     """
 
     @api.multi
+    @api.onchange('is_ytd')
```

Comment:
Why this onchange decorator? I would have thought the depends was enough.

Reply:
It is indeed enough, the depends triggers the onchange, I dropped the decorator in the latest push